e2e-tests/conf/client.yml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: psmdb-client
-          image: percona/percona-server-mongodb:4.0
+          image: percona/percona-server-mongodb:4.4
           imagePullPolicy: Always
```

Comment:
@hors We didn't found any problems to use 4.4 version. Could we meet any problems with it? You might be have more context)